File: URIOJ/uri1895.cpp
/*
 * Leonardo Deliyannis Constantin
 * URI 1895 - Jogo do Limite 
*/

#include <stdio.h>
#include <stdlib.h>

int main(){
    int N, T, L, S, i, dif, v[2];
    while(scanf("%d %d %d", &N, &T, &L) != EOF){
        v[0] = v[1] = 0;
        for(i = 0; i < N-1; i++){
            scanf("%d", &S);
            dif = abs(T-S);
            if(dif <= L){
                T = S;
                v[i&1] += dif;
            }
        }
        printf("%d %d\n", v[0], v[1]);
    }
    return 0;
}

File: URIOJ/uri1895.py
# -*- coding: utf-8 -*-
# Leonardo Deliyannis Constantin
# URI 1895 - Jogo do Limite

def main():
    N, T, L = map(int, input().split())
    v = [0] * 2
    for i in range(N-1):
        S = int(input())
        dif = abs(T-S)
        if dif <= L:
            v[i%2] += dif
            T = S
    print("%d %d" % (v[0], v[1]))
    
if __name__ == '__main__':
    while True:
        try:
            main()
        except EOFError:
            break
